target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -154,6 +154,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "6t@eth0"
 		;;
+	barionet-1000-v1-1-a9)
+		ucidef_add_switch "switch0" \
```

Comment:
Please use `ucidef_set_interface_lan "eth0"` here and merge it with the existing block using `ucidef_set_interface_lan "eth0"? .

There is no need to add a switch config for a single port board. WAN looks wrong to me.